includes/admin/class-sensei-setup-wizard-pages.php

```diff
@@ -184,6 +184,105 @@ private function get_learner_courses_page_template() {
+<div style="padding-bottom:var(--wp--preset--spacing--20)" class="wp-block-sensei-lms-course-categories has-text-align-left"></div>
+<!-- /wp:sensei-lms/course-categories -->
+
+<!-- wp:post-title {"textAlign":"left","isLink":true,"style":{"spacing":{"padding":{"top":"0","right":"0","bottom":"0","left":"0"},"margin":{"top":"0","right":"0","bottom":"0","left":"0"}}}} /-->
+
+<!-- wp:post-author {"textAlign":"left"} /-->
+
+<!-- wp:post-excerpt {"textAlign":"left"} /-->
+
+<!-- wp:sensei-lms/course-progress {"defaultBarColor":"foreground","className":"has-text-align-left"} /--></div>
+<!-- /wp:column -->
+
+<!-- wp:column {"width":"33.33%","style":{"spacing":{"padding":{"top":"0"}}}} -->
+<div class="wp-block-column" style="padding-top:0;flex-basis:33.33%"><!-- wp:sensei-lms/course-actions -->
+<!-- wp:sensei-lms/button-take-course {"align":"right"} -->
+<div class="wp-block-sensei-lms-button-take-course is-style-default wp-block-sensei-button wp-block-button has-text-align-right"><button class="wp-block-button__link">' . __( 'Start Course', 'sensei-lms' ) . '</button></div>
+<!-- /wp:sensei-lms/button-take-course -->
+
+<!-- wp:sensei-lms/button-continue-course {"align":"right"} -->
+<div class="wp-block-sensei-lms-button-continue-course is-style-default wp-block-sensei-button wp-block-button has-text-align-right"><a class="wp-block-button__link">' . __( 'Continue', 'sensei-lms' ) . '</a></div>
+<!-- /wp:sensei-lms/button-continue-course -->
+
+<!-- wp:sensei-lms/button-view-results {"align":"right","className":"is-style-default"} -->
+<div class="wp-block-sensei-lms-button-view-results is-style-default wp-block-sensei-button wp-block-button has-text-align-right"><a class="wp-block-button__link">' . __( 'Visit Results', 'sensei-lms' ) . '</a></div>
+<!-- /wp:sensei-lms/button-view-results -->
+<!-- /wp:sensei-lms/course-actions --></div>
+<!-- /wp:column --></div>
+<!-- /wp:columns --></div>
+<!-- /wp:group -->
+<!-- /wp:post-template -->
+
+<!-- wp:query-pagination {"paginationArrow":"arrow","align":"center","layout":{"type":"flex","justifyContent":"space-between"}} -->
+<!-- wp:query-pagination-previous {"fontSize":"small"} /-->
+
+<!-- wp:query-pagination-numbers /-->
+
+<!-- wp:query-pagination-next {"fontSize":"small"} /-->
+<!-- /wp:query-pagination --></div>',
+						],
+						'attrs'        => [
+							'queryId'       => 0,
+							'query'         => [
+								'postType' => 'course',
+								'perPage'  => 3,
+								'offset'   => 0,
+								'inherit'  => true,
+								'sticky'   => '',
+								'pages'    => 0,
+								'order'    => 'desc',
+								'orderBy'  => 'date',
+								'author'   => '',
+								'search'   => '',
+								'exclude'  => [],
+							],
+							'displayLayout' => [
+								'type' => 'list',
+							],
+							'className'     => 'wp-block-sensei-lms-course-list wp-block-sensei-lms-course-list\u002d\u002dis-list-view',
```

Comment:
There's Unicode characters here too.

Reply:
Thanks! Removed it here https://github.com/Automattic/sensei/pull/6354/commits/6b980060b012015e952515975481922557070a78